dev/doxygen/pages/translations.md

```diff
@@ -0,0 +1,27 @@
+Translation Guidelines {#doc_translations}
+==========================================
+
+[TOC]
+
+# Key Terms {#doc_translations_terms}
+
+For consistent translations, certain LibrePCB specific terms (like "package" or
+"device") should always be translated in the same way.
+
+| **EN** | DE |
+|--------|----|
+| **Pin** | Pin |
+| **Symbol** | Symbol |
+| **Symbol Variant** | Symbol-Variante |
+| **Component** | Komponente |
+| **Device** | Bauteil |
+| **Part** | Typ |
+| **Package** | Gehäuse |
+| **Pad** | Pad |
+| **Footprint** | Footprint |
+| **Workspace** | Workspace |
+| **Project** | Projekt |
+| **Library** | Library |
```

Comment:
I left the translation at "Library" since "Bibliothek" sounds quite stupid. Do you agree?

Reply:
That's a tricky one... IMHO "Bibliothek" is not too bad, but also not very nice. I think we can go with "Library" for the moment - if it turns out another term would be better, we could still change it afterwards.